### ISSUE TITLE
Project status?

### ISSUE THREAD

**Issue by piersdeseilligny** [jupyterlab/lumino#138]
I've been playing around with Lumino for a few days now, and it appears to be an extremely powerful and resilient framework. However it doesn't seem particularly active: there's a decent amount of apparently ignored open issues, and pretty much no documentation, other than what is left from [PhosphorJS](http://phosphorjs.github.io/docs.html) - which could disappear any day.

I was therefore wondering if it would be possible to get an update about the development status of Lumino, and if it would be safe to use in production level software?

**Comment by jasongrout** [comment]
It is being actively maintained (it is the basis for JupyterLab, and shares a lot of the same developers, which does see a lot of development). However, it's very stable, like you mention, and we're also spread quite thin, so you don't see many commits to Lumino itself. We have been pushing forward on accessibility recently, for example, and we recently committed some improvements to its css handling, its command api, etc.

**Comment by jasongrout** [comment]
Of course, we would welcome new contributors. I agree having a Lumino docs and example site would be very helpful.

**Comment by jasongrout** [comment]
>  if it would be safe to use in production level software?

Short answer: we are doing so (in JupyterLab, ipywidgets, and their growing extension ecosystems). Of course you have to make your own judgement call for your production-level software.

**Comment by jasongrout** [comment]
Would you like to help set up a docs and example site like Phosphor has? You might be able to port over phosphor's, or it might be better to make something auto-updating with github actions, etc.

**Comment by vidartf** [comment]
Re docs: there is a video series [here](https://www.youtube.com/playlist?list=PLFx5GKe0BTjQyCKtiK9TI-ekSuSn_8a3J) that might help. Ideally we should write some narrative docs informed by that video series, so if anyone wants to help with that, it would be greatly appreciated!

**Comment by piersdeseilligny** [comment]
Thanks - good to know! And sure I can have a look at setting up some docs, probably using [typedoc](https://github.com/TypeStrong/typedoc). It'll be a good occasion to get 100% accustomed to lumino. Pretty busy these days, but I'il submit a PR if I end up doing so 👍

**Comment by adpatter** [comment]
@vidartf @jasongrout Do you happen to know if anyone has those videos in a format that is easier to see?

**Comment by jasongrout** [comment]
Are you talking about the small font being hard to read? I think some of the later videos we asked Chris to zoom his browser to get bigger fonts, so it was better. Also, I think the videos were recorded at Chris's monitor aspect ratio, which was quite wide. Perhaps if we cropped the videos down it would be easier to read.

CC @isabela-pf who might have a better answer about better resolution copies of the videos.

**Comment by adpatter** [comment]
@isabela-pf I'm interested in learning more about Lumino.  Do you happen to know if there are videos that are easier to see?

**Comment by isabela-pf** [comment]
Hi all! Thanks for your patience with my delayed reply. I don't know of other videos that are easy to see.

I've looked into the posted videos and compared them to the original files since I have access. I think I could crop out most of the blank space on the monitor and have the screen larger (and hopefully easier to read). It might still be somewhat small, but it could be an improvement. I'd have to ask for permission to do this and repost, but I'm happy to ask.

I only have a little video editing experience, so I can also look if there's any other techniques that could help with this mouse-sized text.

**Comment by adpatter** [comment]
@isabela-pf Thank you.  It would be great if we had this resource, due to the limited amount of documentation on Lumino.  If you get permission, you can send one of them to me and I can give it a shot.  If I can figure out how to edit one of them then perhaps I can prepare the others as well.

It looks like the Youtube series is missing the first video that covers the Message class and messaging.
